src/main/java/org/assertj/core/util/DoubleComparator.java

```diff
@@ -23,7 +21,7 @@ public DoubleComparator(double epsilon) {
   }
 
   @Override
-  public int compare(Double x, Double y) {
+  protected int compareNonNull(Double x, Double y) {
```

Comment:
I have created https://github.com/joel-costigliola/assertj-core/issues/1025 for that.